crates/fs/Cargo.toml

```diff
@@ -46,6 +46,7 @@ env_logger = "0.9"
 proptest = "1.0"
 test-log = "0.2"
 test-strategy = "0.2"
+criterion = { version = "0.4.0", features = ["async_std"] }
```

Comment:
more of interesting note: https://www.tweag.io/blog/2022-03-03-criterion-rs/ (criterion can run wasm benchmarks, if you turn default_features off cause rayon)

Reply:
That's great to know but I think it's fine to ignore criterion for wasm, just like we do for the tests and logging. Right now we only test our wasm blob from the Js side and I expect benchmarking will also happen the same way. I think that is a more relevant way of testing everything works from a user perspective. Unless someone has a non-web use case for wnfs, then it is easy to expose what they need behind feature flags.